docker/conf/images/conf-app-images-local.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+FILE_PATH=$(
+  cd "$(dirname "${BASH_SOURCE[0]}")" || exit
+  pwd -P
+)
+
+source "$FILE_PATH"/conf-app-images-original.sh
+
+# ------- image names -------
+
+export ASYNC_RECEIVER_IMAGE=jembi/jempi-async-receiver:0.1.1
+export SYNC_RECEIVER_IMAGE=jembi/jempi-sync-receiver:0.1.1
+export PREPROCESSOR_IMAGE=jembi/jempi-pre-processor:0.1.1
+export CONTROLLER_IMAGE=jembi/jempi-controller:0.1.1
+export EM_IMAGE=jembi/jempi-em-calculator:0.1.1
+export LINKER_IMAGE=jembi/jempi-linker:0.1.1
+export API_IMAGE=jembi/jempi-api:0.1.1
```

Comment:
Every time the version updates we will have to update this. If this is just for development should we not default to building `latest`?

Reply:
Would be nice to have a better way to manage this, but for the sake of local building, I think `latest` is probably fine. It is worth telling the rest of the team to start thinking about version management, so that we can make releases with corresponding Dockerhub releases. For the sake of JeMPI implementations, like in Platform, proper releases are a necessity.